toolip/utils/redoc.py

```diff
@@ -0,0 +1,91 @@
+from dataclasses import dataclass
+from typing import List, Optional, Type, Union
+
+from apispec import APISpec
+from fastapi import APIRouter
+from fastapi.openapi.docs import get_redoc_html
+from pydantic import BaseModel
+from pydantic.schema import schema
+from starlette.responses import JSONResponse
+
+
+@dataclass
+class TagAndModels:
+    models: Union[Type[BaseModel], List[Type[BaseModel]]]
+    tag: Optional[str] = None
+    description: Optional[str] = None
+
+
+def model_to_doc_str(model: Type[BaseModel], title: bool = False):
```

Comment:
I think this is meant to be an internal function. Is it used in any of our apps directly?
```suggestion
def _model_to_doc_str(model: Type[BaseModel], title: bool = False):
```

Reply:
I just checked them and you're right. It's an internal function. Changing it